server/src/filters/sentry.filter.ts

```diff
@@ -0,0 +1,16 @@
+import { Catch, ArgumentsHost, HttpServer } from '@nestjs/common';
+import { AbstractHttpAdapter, BaseExceptionFilter } from '@nestjs/core';
+
+import * as Sentry from '@sentry/node';
+
+@Catch()
+export class SentryFilter extends BaseExceptionFilter {
```

Comment:
@brendanlaschke Should be enough for now, but as @charlesBochet said, when we'll tackle this as a drive other features should be added like the ones in this library: https://github.com/ntegral/nestjs-sentry